.github/README.md

```diff
@@ -1,17 +1,40 @@
 <p align="center">
-  <img alt="Files Logo" src="../src/Files.App%20(Package)/Assets/AppTiles/Release/StoreLogo.scale-400.png" width="100px" />
-  <h1 align="center">Files</h1>
+  <img alt="Files hero image" src="../assets/ReadmeHero.png" />
 </p>
 
-[![Build Status](https://dev.azure.com/filescommunity/Files/_apis/build/status/Build%20Pipeline?branchName=main)](https://dev.azure.com/filescommunity/Files/_build/latest?definitionId=4&branchName=main)
-[![Crowdin](https://badges.crowdin.net/files-app/localized.svg)](https://crowdin.com/project/files-app)
-[![Discord](https://discordapp.com/api/guilds/725513575971684472/widget.png)](https://discord.gg/files)
-[![Download](https://img.shields.io/badge/Download%20Installer-blue.svg?style=flat-round)](https://files.community/download)
-[![Documentation](https://img.shields.io/badge/View%20Documentation-purple.svg?style=flat-round)](https://files.community/docs)
+<p align="center">
+  <a style="text-decoration:none" href="https://files.community/">
+    <img src="https://img.shields.io/badge/Files-Website-F9B81F" alt="Files Website" /></a>
+  <a style="text-decoration:none" href="https://github.com/files-community/Files/actions/workflows/ci.yml">
+    <img src="https://github.com/files-community/Files/actions/workflows/ci.yml/badge.svg" alt="Files CI Status" /></a>
+  <a style="text-decoration:none" href="https://crowdin.com/project/files-app">
+    <img src="https://badges.crowdin.net/files-app/localized.svg" alt="Files Localization Status" /></a>
+  <a style="text-decoration:none" href="https://discord.gg/files">
+    <img src="https://img.shields.io/discord/725513575971684472?label=Discord&color=7289da" alt="Files Discord" /></a>
+</p>
+
+Files is a modern file manager that helps users organize their files and folders. Our mission with Files is to build the best file manager for Windows, and we’re proud to be building it out in the open so everyone can participate. User feedback helps shape the features we work on, & the bug reports on GitHub help to make Files more reliable. Built and maintained by the open-source community, Files features robust multitasking experiences, file tags, deep integrations, and an intuitive design.
+
+- **Community-driven**: Built and maintained by the open-source community.
+- **Intuitive design:** Boasts file management experience with intuitive, sleek, and powerful design built with WinUI.
+- **Multitasking:** The application supports multiple tabs for easy switching between different folders. It also provides a column view for quick file browsing and dual pane support for efficient file management.
+- **File tags:** ????
+- **Integrations:** The project integrates with QuickLook for previewing files without opening them, and users can customize the background color to match their personal style.
```

Comment:
We might not need those?

Reply:
Correct, we don't need this in the readme.